terra_notebook_utils/__init__.py

```diff
@@ -5,6 +5,7 @@
 WORKSPACE_GOOGLE_PROJECT = os.environ.get('GOOGLE_PROJECT')  # This env var is set in Terra Cloud Environments
 TERRA_DEPLOYMENT_ENV = os.environ.get('TERRA_DEPLOYMENT_ENV', 'prod')
 MARTHA_URL_VERSION = os.environ.get('MARTHA_URL_VERSION', 'martha_v3')
+DRS_RESOLVER = os.environ.get('DRS_RESOLVER', 'martha')  # values for DRS_RESOLVER are either 'drshub' or 'martha'
```

Comment:
Should this default to `'drshub'` now?

Reply:
Since we haven't fully perf tested DRSHub yet, I think it should stay as Martha for now. However as soon as DRSHub is ready for traffic it should definitely default to DRSHub. Do you want me to make a ticket on my board to switch that when it's ready?

Reply:
Yes please, thanks!

Reply:
Cool, I'll do that right now, then it won't get lost in the shuffle!